examples/with-svelte/src/App.svelte

```diff
@@ -1,27 +1,33 @@
 <script lang="ts">
   import { cropImage, type ZoomImageWheelState } from "@zoom-image/core"
   import { useZoomImageClick, useZoomImageHover, useZoomImageMove, useZoomImageWheel } from "@zoom-image/svelte"
-  import { tick } from "svelte"
+  import { onMount, tick } from "svelte"
 
-  const tabs: {
+  let tabs: {
     name: string
     href: string
     current: boolean
     value: "wheel" | "hover" | "move" | "click"
-  }[] = [
+  }[] = $state([
     { name: "Wheel", href: "#", current: true, value: "wheel" },
     { name: "Hover", href: "#", current: false, value: "hover" },
     { name: "Move", href: "#", current: false, value: "move" },
     { name: "Click", href: "#", current: false, value: "click" },
-  ]
-  $: zoomType = tabs.find((tab) => tab.current)?.value as "wheel" | "hover" | "move" | "click"
-  let croppedImage: string = ""
+  ])
+  let zoomType = $derived(tabs.find((tab) => tab.current)!.value)
+  let croppedImage = $state("")
 
+  // svelte-ignore non_reactive_update
```

Comment:
FYI you technically don't even *need* to migrate to runes just yet, and if the entire file doesn't use `$state`/`$derived`/etc it shouldn't error I think

Reply:
Yeah. It still runs fine since Svelte is a compiler instead of runtime like React. SinceI am working full-time at Shopify where we have the largest TypeScript React codebase in the world 🙈 - I wanted to give Svelte 5 a try following [this guide](https://svelte.dev/docs/svelte/v5-migration-guide) 😄

Reply:
Gotcha! Svelte is great :D

Reply:
Thanks Daniel. I think I will add a separate example for Svelte 5 instead of rewriting everything. Since people still use Svelte 4 with magical syntax `$` 😃